app/views/unattended/provisioning_templates/provision/kickstart_default.erb

```diff
@@ -187,11 +187,8 @@ authselect --useshadow --passalgo=<%= @host.operatingsystem.password_hash.downca
 <% else -%>
 authconfig --useshadow --passalgo=<%= @host.operatingsystem.password_hash.downcase || 'sha256' %> --kickstart
 <% end -%>
-<% if use_ntp -%>
-timezone --utc <%= host_param('time-zone') || 'UTC' %>
-<% else -%>
+
```

Comment:
Please remove this empty line to make tests passing, we have snapshots which are compared to rendered content and it's currently failing:

```
Diff for /home/jenkins/workspace/test_develop_pr_core/database/postgresql/ruby/2.5/slave/fast/test/unit/foreman/renderer/snapshots/ProvisioningTemplate/provision/Kickstart default.snap.txt:
--- expected
+++ actual
@@ -18,6 +18,7 @@
 rootpw --iscrypted xybxa6JUkz63w
 firewall --service=ssh
 authconfig --useshadow --passalgo=sha256 --kickstart
+
 timezone --utc UTC 
```

Reply:
It has been removed with a new commit.